src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PersonEntity.kt

```diff
@@ -93,6 +93,9 @@ class PersonEntity(
   @Column(name = "date_of_birth")
   var dateOfBirth: LocalDate? = null,
 
+  @Column(name = "date_of_birth_text")
```

Comment:
Add a text column for date, as you cant have a gin index on date data type. As well as not allowing 'to_char' function when creating a index. So best option to store as text as well and run fuzzy search on that column instead

Reply:
Yeah good shout, i'll monitor the results of this to see if its worth keeping. If so, ill swap to view